src/main/java/org/sigmah/server/dao/impl/MonitoredPointHibernateDAO.java

```diff
@@ -33,4 +38,23 @@
  */
 public class MonitoredPointHibernateDAO extends AbstractDAO<MonitoredPoint, Integer> implements MonitoredPointDAO {
 
+	@Override
+	public List<MonitoredPoint> findNotCompletedByProjectIds(Set<Integer> projectIds) {
+		if (projectIds == null || projectIds.isEmpty()) {
```

Comment:
Remove `projectIds == null || `, should never occur (I've checked the call and collection passed is never null, we should always expect empty collection is favored over null as a global rule of thumb).